tests/ethpm/test_package_init_from_ipfs.py

```diff
@@ -1,26 +1,18 @@
+import os
+
 import pytest
 
-import ethpm
 from ethpm import (
     Package,
 )
 
-VALID_IPFS_PKG = 'ipfs://QmeD2s7KaBUoGYTP1eutHBmBkMMMoycdfiyGMx2DKrWXyV'
-
-
-# mock out http req to IPFS gateway
-# `fetch_ipfs_package` returns local 'safe-math-lib` pkg
-@pytest.fixture(autouse=True)
-def mock_request(monkeypatch, safe_math_manifest):
-    def mock_fetch(x):
-        return safe_math_manifest
-    monkeypatch.setattr(ethpm.package, 'fetch_ipfs_package', mock_fetch)
-
 
 def test_package_from_ipfs_with_valid_uri():
-    package = Package.from_ipfs(VALID_IPFS_PKG)
+    os.environ['URI_BACKEND_CLASS'] = 'ethpm.backends.ipfs.DummyIPFSBackend'
+    package = Package.from_ipfs('safe-math-lib/1.0.0.json')
     assert package.name == 'safe-math-lib'
     assert isinstance(package, Package)
+    os.environ.pop('URI_BACKEND_CLASS')
```

Comment:
You should use [`monkeypatch.setenv`](https://docs.pytest.org/en/latest/monkeypatch.html) to ensure that things get cleaned up.